src/BPEssentials/FileChecker.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace BPEssentials
+{
+    public static class FileChecker
+    {
+        public static readonly HttpClient client = new HttpClient();
```

Comment:
Suggested to only use properties and private fields. Consider changing to a property (getter only).

```suggestion
        public static HttpClient Client { get; }= new HttpClient();
```